src/components/axes/timeAxis.ts

```diff
@@ -469,7 +513,8 @@ export module Axis {
       var visibleTickLabels = this._tierLabelContainers[index]
                                     .selectAll("." + AbstractAxis.TICK_LABEL_CLASS)
                                     .filter(function(d: Element, i: number) {
-                                      return d3.select(this).style("visibility") === "visible";
+                                      var visibility = d3.select(this).style("visibility");
+                                      return visibility === "visible" || visibility === "inherit";
```

Comment:
If the parent visibility is not visible 
```typescript
d3.select(this).style("visibility");
```
Returns "hidden" in everything except IE9. For IE9, I could not figure out what is the rule, nor I could find resources to tell me. It seems like sometimes it correctly computes the visibility, other times it does not. This filter is an optimization anyway, so it will work fine, even if more elements are returned.